### ISSUE TITLE
Problem with unzipping the data file

### ISSUE THREAD

**Issue by coolmaksat** [yourh/DeepGraphGO#1]
Hi,
I'm trying to unzip the data file but getting some errors.
Could you please help or provide concatenated zip file. 

Thank you.

**Comment by yourh** [comment]
Did you download all files under the data folder?

**Comment by coolmaksat** [comment]
Hi,
yes, I did with 'git clone'.
data.z[01-06] and data.zip
This is what I get when I try to unzip.
Archive:  data.zip
warning [data.zip]:  zipfile claims to be last disk of a multi-part archive;
  attempting to process anyway, assuming all parts have been concatenated
  together in order.  Expect "errors" and warnings...true multi-part support
  doesn't exist yet (coming soon).
file #1:  bad zipfile offset (local header sig):  4
file #2:  bad zipfile offset (local header sig):  257014
file #3:  bad zipfile offset (local header sig):  796021
file #4:  bad zipfile offset (local header sig):  927401
file #5:  bad zipfile offset (local header sig):  933560
file #6:  bad zipfile offset (local header sig):  1280827
file #7:  bad zipfile offset (local header sig):  1327275
file #8:  bad zipfile offset (local header sig):  1332097
file #9:  bad zipfile offset (local header sig):  24897202
file #10:  bad zipfile offset (local header sig):  27849366
file #11:  bad zipfile offset (local header sig):  28077466
file #12:  bad zipfile offset (local header sig):  28386157
file #13:  bad zipfile offset (local header sig):  28418537
file #14:  bad zipfile offset (local header sig):  28422186
file #15:  bad zipfile offset (local header sig):  28538584
file #16:  bad zipfile offset (local header sig):  28550697
file #17:  bad zipfile offset (local header sig):  28552456
file #18:  bad zipfile offset (local header sig):  44545432
file #19:  bad zipfile offset (local header sig):  46371598
file #20:  bad zipfile offset (local header sig):  46530173
file #21:  bad zipfile offset (local header sig):  46685192
file #22:  bad zipfile offset (local header sig):  46699236
file #23:  bad zipfile offset (local header sig):  46701245
file #24:  bad zipfile offset (local header sig):  22113326
file #25:  bad zipfile offset (local header sig):  23462683
file #26:  bad zipfile offset (local header sig):  24518807
file #27:  bad zipfile offset (lseek):  74645504
file #28:  bad zipfile offset (lseek):  77291520

I also tried to concatenate the files and then unzip, but it did not help.
